#! /bin/sh /usr/share/dpatch/dpatch-run
## 04-change-md5sum.dpatch by Tiago Saboga <tiagosaboga@gmail.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Jna fails to build when it detects a change in the headers 
## DP: of the native library, but it relies on the output of javah to
## DP: compute a md5sum, and javah from gcj produces different output.

@DPATCH@
diff -urNad libjna-java-3.0.2~/build.xml libjna-java-3.0.2/build.xml
--- libjna-java-3.0.2~/build.xml	2008-02-28 16:34:44.000000000 -0300
+++ libjna-java-3.0.2/build.xml	2008-05-23 18:50:49.000000000 -0300
@@ -40,7 +40,7 @@
   <property name="jni.revision" value="3"/>
   <property name="jni.build" value="${build.number}"/>
   <property name="jni.version" value="${jni.major}.${jni.minor}.${jni.revision}"/>
-  <property name="jni.md5" value="892beacd437514d23ed9b1cefeb2ead6"/>
+  <property name="jni.md5" value="3a865a03042a115149114fc8e01bd9c8"/>
   <property name="spec.title" value="Java Native Access (JNA)"/>
   <property name="spec.vendor" value="${vendor}"/>
   <property name="spec.version" value="${jna.major}"/>
